packages/components/components/elvis-toolbox/src/hooks/useLanguage.ts

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from 'react';
+
+export const useLang = () => {
+  const [lang, setLang] = useState<'no' | 'en'>('no');
```

Comment:
Extract the type `'no' | 'en'` to a type `Language` (or something like that) so it can be reused in components, as I see that you're redefining it in some places in pagination 🧹